lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APrioriLinear.java

```diff
@@ -381,9 +380,6 @@ public APrioriLinear(
             }
             log.info("Time spent in order {}:  {} ms", curOrderFinal, System.currentTimeMillis() - startTime);
         }
-        log.info("Time spent in APriori:  {} ms", System.currentTimeMillis() - beginTime);
```

Comment:
Why delete this and line 94? This is useful for isolating the Apriori time from the encoding time

Reply:
Good call, fixed